source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -48,4 +67,687 @@ namespace AdaptiveNamespace
             jsonObject, elementParserRegistration, actionParserRegistration, adaptiveWarnings, element);
     }
     CATCH_RETURN;
+
+    HRESULT XamlBuilder::BuildImage(_In_ IAdaptiveCardElement* adaptiveCardElement,
+                                    _In_ IAdaptiveRenderContext* renderContext,
+                                    _In_ IAdaptiveRenderArgs* renderArgs,
+                                    _COM_Outptr_ IUIElement** imageControl)
```

Comment:
This makes sense, but is a little tricky. Let's stick a comment on this explaining why we're defining the XamlBuilder method in the ImageRenderer code file.